src/tcgwars/logic/impl/gen8/RebelClash.groovy

```diff
@@ -2031,10 +2031,10 @@ public enum RebelClash implements LogicCardInfo {
         resistance F, MINUS30
         bwAbility "Counterattack", {
           text "If this Pokemon is your Active Pokemon and is damaged by an opponent’s attack, place 3 damage counters on the attacking Pokemon."
-          delayedA {
+          delayedA (priority: LAST) {
             before APPLY_ATTACK_DAMAGES, {
               if (bg.currentTurn == self.owner.opposite && bg.dm().find({ it.to==self && it.dmg.value }) && self.active) {
-                powerused()
+                bc "Counterattack activates"
```

Comment:
Why this change? Without a check for last turn usage, it should produce this same output.